scipy/spatial/qhull.pyi

```diff
@@ -0,0 +1,98 @@
+'''
+Static type checking stub file for scipy/spatial/qhull.pyx
+'''
+
+from typing import TYPE_CHECKING, List, Tuple, Any
+
+if TYPE_CHECKING:
+    from numpy.typing import ArrayLike
+else:
+    ArrayLike = Any
+
+import numpy as np
+
+class _Qhull:
+    def __init__(self, points: ArrayLike, furthest_site: bool = ...,
+                 incremental: bool = ..., qhull_options=...): ...
```

Comment:
`qhull_options` should be `str` I think.

Reply:
I did try `str`, but it's converted later to `bytes` which mypy will complain about and I'm not mypy-savy enough to figure out how to get it working